src/main/java/com/minecolonies/coremod/colony/requestsystem/data/StandardRequestSystemBuildingDataStore.java

```diff
@@ -0,0 +1,212 @@
+package com.minecolonies.coremod.colony.requestsystem.data;
+
+import com.google.common.reflect.TypeToken;
+import com.minecolonies.api.colony.requestsystem.StandardFactoryController;
+import com.minecolonies.api.colony.requestsystem.data.IRequestSystemBuildingDataStore;
+import com.minecolonies.api.colony.requestsystem.factory.FactoryVoidInput;
+import com.minecolonies.api.colony.requestsystem.factory.IFactory;
+import com.minecolonies.api.colony.requestsystem.factory.IFactoryController;
+import com.minecolonies.api.colony.requestsystem.token.IToken;
+import com.minecolonies.api.util.NBTUtils;
+import com.minecolonies.api.util.constant.TypeConstants;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.util.Tuple;
+import net.minecraftforge.common.util.Constants;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static com.minecolonies.api.util.constant.NbtTagConstants.*;
+
+public class StandardRequestSystemBuildingDataStore implements IRequestSystemBuildingDataStore
+{
+
+    private IToken<?>                                id;
+    private final Map<TypeToken<?>, Collection<IToken<?>>> openRequestsByRequestableType;
+    private final Map<Integer, Collection<IToken<?>>>      openRequestsByCitizen;
+    private final Map<Integer, Collection<IToken<?>>>      completedRequestsByCitizen;
+    private final Map<IToken<?>, Integer>                  citizenByOpenRequest;
+
+    public StandardRequestSystemBuildingDataStore(
+      final IToken<?> id,
+      final Map<TypeToken<?>, Collection<IToken<?>>> openRequestsByRequestableType,
+      final Map<Integer, Collection<IToken<?>>> openRequestsByCitizen,
+      final Map<Integer, Collection<IToken<?>>> completedRequestsByCitizen,
+      final Map<IToken<?>, Integer> citizenByOpenRequest) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)